tests/factory.py

```diff
@@ -15,3 +16,18 @@ def insertDummyLanguages():
         for entry in data:
             Languages(language=entry["language"], code=entry["code"],
                       type=entry["type"], default=entry["default"]).save()
+
+
+def insertDummySentences():
+    insertDummyLanguages()
+    lang_fr = Languages.objects(language="francais").first()
+    lang_en = Languages.objects(language="english").first()
+    lang = lang_fr
+    with open('db/dummy/sentences.json') as f:
+        data = json.load(f)
+        for entry in data:
+            if entry["language"] == "francais":
```

Comment:
are languages always stored in lower case? In order to be more robust, you could convert both strings to lower cases before making comparisons.

Reply:
There is a way of writing an equivalent code without the if. you basically first create a map from string -> language eg : map: "francais" -> Languages.objects(language="francais").first() then in the while loop you do map[entry["language"]]. You still need to be careful about case sensitive strings.

Reply:
I don't know how to use `map`, I will look into it.